packages/transformers/svg-react/src/SvgReactTransformer.js

```diff
@@ -34,6 +34,7 @@ export default (new Transformer({
     );
 
     asset.type = 'jsx';
+    asset.bundleBehavior = 'inline';
```

Comment:
I think this should be `null` instead. `inline` means it will create a separate bundle, and then inline it, but we really just want it to be in the same bundle as the parent javascript.

```suggestion
    asset.bundleBehavior = null;
```